lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/project/ExternalS3UploadDestinationLocationSetting.json

```diff
@@ -0,0 +1,22 @@
+{
+	"description": "The external upload destination setting contains information to generate an SFTP or HTTPS upload destination.",
+	"implements": [
+		{
+			"$ref": "org.sagebionetworks.repo.model.project.UploadDestinationLocationSetting"
+		}
+	],
+	"properties": {
+		"url": {

```

Comment:
This is really the S3 service endpoint needed to connect to S3 (defaults to east).  The docs should include a link to the AWS docs for picking this endpoint.